src/cli.js

```diff
@@ -215,6 +215,7 @@ async function runCmd (argv, stdout, stderr) {
       let startTime = Date.now();
       let ps;
       const buildFile = eval("require.resolve")(resolve(args._[1] || "."));
+      const ext = buildFile.endsWith('.cjs') ? '.cjs' : '.js';
```

Comment:
Let's create a function so this works with `.mjs` too.

```js
function getOutputExtension(input: string) {
  const ext = path.extname(input);
  if (ext === '.mjs' || ext === '.cjs') {
    return ext;
  }
  return '.js';
}
```

Then it can be reused in both places.

Reply:
As mentioned above - yes this can easily extend to `.mjs`, but requires the support for an ES module output first, which should probably be a separate flag in its own PR.